### PR TITLE
Extend deprecation notice

### PR DESCRIPTION
**Pull request by derrabus**
See https://github.com/doctrine/annotations/issues/485#issuecomment-1676447349

### CODE REVIEW

**Review by greg0ire — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ which are a native replacement for annotations. As such, this library is
 considered feature complete, and should receive exclusively bugfixes and
 security fixes.
 
+We do not recommend using this library in new projects and encourage authors
```
Comment (malarzm) [inline README.md:7]: Let's add `# Project deprecated` or something like this at the top so there's something big screaming "don't use me" :) 

Also maybe a link to our blogpost https://www.doctrine-project.org/2022/11/04/annotations-to-attributes.html like "This is how Doctrine handled migration" to help folks discover this?

Comment (derrabus) [reply README.md:7]: I think, the ⚠️ is enough in that regard. I've added a link to the blog.

**Comment by derrabus** [comment]
I completely forgot about this one. Shall we merge the change before we make a bugfix release?

@malarzm @greg0ire

**Comment by greg0ire** [comment]
Sure, it just touches documentation so…

**Pull request merged**